<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ResourceManager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>System info</string>
       </attribute>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>210</y>
          <width>671</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>CPU INFO</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CPU name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_cpu_name">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>50</y>
          <width>431</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>90</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CPU type:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_cpu_type">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>90</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>130</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CPU frequency:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_cpu_fre">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>130</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>170</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Cache capacity:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_cache">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>170</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>220</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>OS INFO</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>270</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>OS type:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>330</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>OS version:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>380</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>GCC version:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_os_type">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>270</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_os_version">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>330</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_GCC">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>380</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Process info</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>-5</x>
          <y>1</y>
          <width>676</width>
          <height>451</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Performance</string>
       </attribute>
       <widget class="QProgressBar" name="progressBar_mem">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>60</y>
          <width>481</width>
          <height>25</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>220</y>
          <width>681</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>171</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Memory and Swap </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>67</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Memory:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>41</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>used:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_mem_used">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>91</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>100</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>available:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_mem_available">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>100</y>
          <width>111</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>100</y>
          <width>41</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>total:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_mem_total">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>100</y>
          <width>161</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>150</y>
          <width>41</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Swap:</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_swap">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>481</width>
          <height>25</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_swap_used">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>190</y>
          <width>91</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_swap_available">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>190</y>
          <width>111</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_swap_total">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>190</y>
          <width>161</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>190</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>available:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>190</y>
          <width>41</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>total:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>190</y>
          <width>41</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>used:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>310</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>CPU</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_cpu">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>290</y>
          <width>241</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(237, 212, 0);</string>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>410</y>
          <width>121</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>stop monitor</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>About</string>
       </attribute>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>601</width>
          <height>111</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 57 italic 15pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>Design for OS_Course,HuaZhong University of Science and technology</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>390</y>
          <width>211</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>designed by HanZeyan</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>420</y>
          <width>211</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>2020.7</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
